hcatalog/webhcat/svr/src/main/java/org/apache/hive/hcatalog/templeton/tool/HDFSCleanup.java

```diff
@@ -134,7 +132,7 @@ public void run() {
    * @throws IOException
    */
   private void checkFiles(FileSystem fs) throws IOException {
-    long now = new Date().getTime();
+    long now = System.currentTimeMillis();
```

Comment:
This one uses the current time to compare to a files last modified time, so `System.nanoTime();` doesn't work here